code/modules/mob/living/announcer.dm

```diff
@@ -0,0 +1,111 @@
+// This mob is a lie. Similar to /mob/abstract it's not really supposed to do much.
+// It is used as a fake announcer so that we have a lightweight mob to pass to radio broadcasts
+
+/mob/living/announcer
+	name = "Announcer"
+	voice_name = "synthesized voice"
+	accent = ACCENT_TTS
+
+	icon = 'icons/mob/AI.dmi'
+	icon_state = "ai"
+	gender = NEUTER
+
+	status_flags = GODMODE|NO_ANTAG|NOFALL
+	invisibility = INVISIBILITY_ABSTRACT
+	mob_thinks = FALSE
+
+	universal_understand = TRUE
+	can_have_vision_cone = FALSE
+	tesla_ignore = TRUE
+	stop_sight_update = TRUE
+	density = FALSE
+	burn_mod = 0
+	brute_mod = 0
+
+/mob/living/announcer/Initialize()
+	// we specifically do not call parent Initialize here because this mob should not need it and its point is to be lightweight
+	if(initialized)
+		crash_with("Warning: [src]([type]) initialized multiple times!")
+	initialized = TRUE
+
+	for(var/K in all_languages)
+		add_language(K)
+	default_language = all_languages[LANGUAGE_TCB]
+
+/mob/living/announcer/Destroy()
+	for(var/C in contents)
+		qdel(C)
+	QDEL_NULL(reagents)
+	return ..()
```

Comment:
All of this happens in the parent, why duplicate it if you're doing `return ..()` ?

Reply:
Good catch; I'm not sure where but I think I've seen the parents remove the contents in a way that makes it fall on the floor instead of deleting it, but I can't find that now.

TBF I could probably get away with just returning without calling parent here.